src/draw/lv_draw_buf.c

```diff
@@ -84,37 +84,32 @@ lv_draw_buf_handlers_t * lv_draw_buf_get_handlers(void)
 
 uint32_t lv_draw_buf_width_to_stride(uint32_t w, lv_color_format_t color_format)
 {
-    return lv_draw_buf_width_to_stride_user(&default_handlers, w, color_format);
+    return lv_draw_buf_width_to_stride_ex(&default_handlers, w, color_format);
```

Comment:
renaming stuff I think we can open a new pr to process
here just change the code logic

Reply:
I have separated the changes to two commits.